.eslintrc.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
- [ ] This is a pair-programming project but i cannot see any commit from your partner's side. Kindly ask your partner to make some contributions in this pull request :blush: